openff/utilities/provenance.py

```diff
@@ -10,20 +10,30 @@ def _get_conda_list_package_versions() -> Dict[str, str]:
     from openff.utilities.exceptions import CondaExecutableNotFoundError
     from openff.utilities.utilities import has_executable
 
+    used_micromamba = False
     if has_executable("mamba"):
         conda_executable = "mamba"
     elif has_executable("conda"):
         conda_executable = "conda"
     elif has_executable("micromamba"):
         conda_executable = "micromamba"
+        used_micromamba = True
     else:
         raise CondaExecutableNotFoundError()
 
     output = subprocess.check_output([conda_executable, "list"]).decode().split("\n")
 
     package_versions = {}
 
-    for output_line in output[3:-1]:
+    # The output format of `conda`/`mamba list` and `micromamba list` are different.
+    # See https://github.com/openforcefield/openff-utilities/issues/65
+    if used_micromamba:
```

Comment:
Could we just do
```suggestion
    if conda_executable == "micromamba":
```

so we don't have to keep track of that boolean above?

Reply:
Oh yeah, that's way simpler. Implemented in my last commit. Thanks for the suggestion, I'll leave merging in your hands :-)